source/rust_verify_test/tests/integer_ring.rs

```diff
@@ -7,9 +7,9 @@ use common::*;
 test_verify_one_file! {
```

Comment:
It would be nice, but not critical if some of the tests checked for unsupported features and how we emit the error message for those (I only see new tests with `assert_one_fails`).

Reply:
There are a few tests on unsupported features, e.g. https://github.com/verus-lang/verus/blob/edc5773030dd9b621fd3a17271fff22d33e53a75/source/rust_verify_test/tests/integer_ring.rs#L266
But they are written in a way similar to other expected-verification-failure tests. 
Is there a standard way to test and differentiate unsupported error vs. verification error?

Reply:
We generally assert that what we're getting is a `VIR` error message, e.g. in https://github.com/verus-lang/verus/blob/eb988fe4a8600e012822fa0d6b0b5d94a14f0c23/source/rust_verify_test/tests/unions.rs#L77
and that the message matches what we expect.

Reply:
You're also welcome to write a variation of the `assert_one_fails` function that accounts for `not supported` and does the appropriate matching.